docs/manual.md

```diff
@@ -614,6 +615,30 @@ mount:
     filesystem: xfs
 ```
 
+### matching
```

Comment:
Can you provide an example with templates for the readme? or a brief explanation like you did on this PR.